#!/usr/bin/env python
PACKAGE = "cam_rad_fusion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("calib_xt", double_t, 0, "calib_xt", 0.0, -1.00, +1.00)
gen.add("calib_yt", double_t, 0, "calib_yt", 0.0, -1.00, +1.00)
gen.add("calib_zt", double_t, 0, "calib_zt", 0.0, -1.00, +1.00)
gen.add("calib_r",  double_t, 0, "calib_r",  0.0, -180.0, +180.0)
gen.add("calib_p",  double_t, 0, "calib_p",  0.0, -180.0, +180.0)
gen.add("calib_y",  double_t, 0, "calib_y",  0.0, -180.0, +180.0)

exit(gen.generate(PACKAGE, "dynamic_reconfigure", "dyn_reconfig"))